package.json

```diff
@@ -20,10 +20,14 @@
   "repository": "nanostores/preact",
   "sideEffects": false,
   "type": "module",
-  "types": "./index.d.ts",
+  "types": "./src/index.d.ts",
```

Comment:
We don’t use `src` in any Nano Stores projects, so let’s keep code style.

If you want a separated file, just create `./signals/index.js`, etc

Reply:
Okay, I'll fix the structure.